service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -206,8 +205,9 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
 
         data.put("orderCollection", orders);
 
-        if (featureToggleService.isCloseCaseEnabled() && caseData.getOrderTypeAndDocument().getSubtype() != INTERIM) {
-            List<Element<Child>> updatedChildren = getUpdatedChildren(caseData);
+        if (featureToggleService.isCloseCaseEnabled() && caseData.getOrderTypeAndDocument().isClosable()) {
```

Comment:
I think this logic leaks a bit to the controller... I think it should be service responsibility to determine if the order should be closed, not controller! 

But I am not massively concerned... It's not like we aree going to have different entry point - but the controller test, testing logic are a bit fishy.